extensions/wikia/CreateNewWiki/CreateNewWikiController.class.php

```diff
@@ -450,14 +443,6 @@ private function setUserNotLoggedInErrorResponse() {
 		$this->response->setVal( self::STATUS_HEADER_FIELD, wfMessage( 'cnw-error-anon-user-header' )->text() );
 	}
 
-	private function setEmailNotConfirmedErrorResponse() {
-		$this->warning("CreateWiki: user's email not confirmed" );
-		$this->response->setCode( 403 );
-		$this->response->setVal( self::STATUS_FIELD, self::STATUS_ERROR );
-		$this->response->setVal( self::STATUS_MSG_FIELD, wfMessage( 'cnw-error-unconfirmed-email' )->parse() );
-		$this->response->setVal( self::STATUS_HEADER_FIELD, wfMessage( 'cnw-error-unconfirmed-email-header' )->text() );
```

Comment:
I'm sure the SUS team knows the particulars here, but I'm wondering if these i18n keys should be removed since they are no longer being used.

Reply:
good call, removed in 6b24629